packages/components-vue/src/shims/shims-vue.d.ts

```diff
@@ -0,0 +1 @@
+declare module '*.vue';
```

Comment:
Why would we need this now all of a sudden? 
A comment would be helpful if this is still relevant.